doc/man3/RAND_get0_primary.pod

```diff
@@ -28,6 +34,12 @@ directly, but is used internally to reseed the other two instances.
 
 These functions here provide access to the shared DRBG instances.
 
+The three functions all the default arrangement of DRBGs to be modified to
```

Comment:
You may want to reread this ..

Reply:
What waffle I typed, and read and spell checked.
Rewritten now.